assets/js/view/company.rb

```diff
@@ -86,14 +87,62 @@ def render
         ]),
       ]
 
-      if @bids
+      if @bids&.length&.positive?
         children << h(:div, { style: bidders_style }, 'Bidders:')
         children << render_bidders
       end
 
       children << h(:div, { style: bidders_style }, "Owner: #{@company.owner.name}") if @company.owner
 
-      h(:div, props, children)
+      h(:div, [
```

Comment:
why do this, add it to the children instead

Reply:
so that it's outside of the card.

Reply:
if I add to the children the action will appear within the card

Reply:
i see